CHANGES.rst

```diff
@@ -53,7 +53,7 @@ associations
 
 - Updated NIRSpec Lamp calibrations to be grating-specific. [#2780]
 
-- Added new NIRSpec target acq exposure types NRS_WATA and NRS_MSATA. [#2780]
+- Added new NIRSpec target acq exposure types NRS_WATA and NRS_MSATA. [#2780, #2781]
```

Comment:
This looks like it's being added to the associations section of the change log, but it belongs in the assign_wcs section.